scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -468,9 +499,14 @@ while [[ "$running" != "true" ]]; do
   running=$(is_nginx_running "$nginxContainerId")
 done
 
-docker exec "$nginxContainerId" bash -c "curl -s -o /etc/nginx/private/cert.pem https://local-ip.medicmobile.org/fullchain"  2>/dev/null
-docker exec "$nginxContainerId" bash -c "curl -s -o /etc/nginx/private/key.pem https://local-ip.medicmobile.org/key"  2>/dev/null
-docker exec "$nginxContainerId" bash -c "nginx -s reload"  2>/dev/null
+# output a new line with echo and then update certs every time we run to ensure they're current
+echo;update_nginx_local_ip_tls_cert "$nginxContainerId"
+if [ "$(validate_tls "$projectURL")" ];then
```

Comment:
```suggestion
if [ "$(validate_tls "$projectURL" 0)" ];then
```

don't forget the initial value for `$attempt` :eyes:

Reply:
yikes - thanks!